dropwizard2/src/main/java/com/linecorp/armeria/dropwizard/ArmeriaServerFactory.java

```diff
@@ -117,7 +117,8 @@ private void addDefaultHandlers(Server server, Environment environment, MetricRe
                 environment.getJerseyServletContainer(),
                 metrics);
         final Handler adminHandler = createAdminServlet(server, environment.getAdminContext(),
-                                                        metrics, environment.healthChecks());
+                                                        metrics, environment.healthChecks(),
+                                                        environment.admin());
```

Comment:
this will fix for drop wizard 2 but will fail drop wizard 1

Reply:
Should we use reflections to check whether the method is available?
For example:
https://github.com/line/armeria/blob/871d87297e4d051241589cb1ae95641cbc83f880/jetty9/src/main/java/com/linecorp/armeria/server/jetty/JettyService.java#L84-L88

Reply:
@ikhoon thanks, i've migrated to using reflection (but just not using method handle given it's one time call).